chia/cmds/wallet.py

```diff
@@ -316,13 +308,12 @@ def get_address_cmd(wallet_rpc_port: Optional[int], id, fingerprint: int, new_ad
 @click.option(
     "-m", "--fee", help="A fee to add to the offer when it gets taken, in XCH", default="0", show_default=True
 )
-def clawback(wallet_rpc_port: Optional[int], id, fingerprint: int, tx_ids: str, fee: str) -> None:  # pragma: no cover
-    extra_params = {"id": id, "tx_ids": tx_ids, "fee": fee}
-    import asyncio
-
+def clawback(
```

Comment:
maybe I'm going overkill with suggesting `*,` on these functions. if you think it makes sense, feel free to add them

Reply:
it cant hurt

Reply:
added